webapp/src/main/webapp/iaf/frankdoc/views/attribute-description.html

```diff
@@ -0,0 +1,20 @@
+<div>
+  <div ng-if="! attr.enum">{{ attr.description }}</div>
+  <div ng-if="attr.enum">
+    {{ attr.description }}
```

Comment:
vergeet niet de description als `ng-bind-html` op te nemen. Nu gaat functionaliteit verloren.

Reply:
Dit heb ik proberen te doen. Ik ben benieuwd of het klopt en of ik goed begrijp waar dit voor is. Het gevaar is dat een hacker een valse JSON file kan leveren voor deze frontend. Of dat in onze Java code JavaDocs staan met gevaarlijke HTML. De description komt uit een javadoc comment en kan zelf dus HTML code bevatten. We willen die HTML corrigeren zodat gevaarlijke links en scripts onschadelijk worden gemaakt. De website moet die onschadelijk gemaakte HTML tonen, niet de letterlijke tekst die in de JSON staat.